infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/infrastructure/plugins/DefaultClusterManager.java

```diff
@@ -69,15 +86,27 @@ public class DefaultClusterManager extends CorfuReplicationClusterManagerBaseAda
     private String corfuEndpoint = "localhost:9000";
 
     private DefaultClusterConfig topology;
+    private final Map<ClusterConfigurationMsg, Set<LogReplication.ReplicationModel>> remoteSourcesToReplicationModels;
+    private final Map<ClusterConfigurationMsg, Set<LogReplication.ReplicationModel>> remoteSinkToReplicationModels;
+    private final Set<LogReplicationClusterInfo.ClusterConfigurationMsg> connectionEndPoints;
+    private boolean topologyBucketsIinitialized;
```

Comment:
can you add a comment explaining what this variable means?

Same for the below 2 variables as well.